test/integration/datatypes-in-results-test.js

```diff
@@ -523,6 +527,25 @@ exports.variantDateTimeOffset = function(test) {
   );
 };
 
+
+exports.variantMultipleDatatypes = function(test) {
+  const sql = `\
+  create table #tab1 (
+  [c0] [int] IDENTITY(1,1),
+  [c1] [sql_variant] NULL);
+  insert into #tab1 ([c1]) values (N'abcdШ');
+  insert into #tab1 ([c1]) select cast(3148.29 as decimal(20,8));	
+  insert into #tab1 ([c1]) select cast(0x1234 as varbinary(16));
+  insert into #tab1 ([c1]) select cast('01234567-89AB-CDEF-0123-456789ABCDEF' as uniqueidentifier); 
+  insert into #tab1 ([c1]) values (0.00000090000000000);	--decimal(38,17);
+  insert into #tab1 ([c1]) select cast('2011-12-4 10:04:23' as datetime);
+  insert into #tab1 ([c1]) select cast('abcde' as varchar(10));
+  select [c1] from #tab1 ORDER BY [c0];
+  `;
+  const expectedValues = ['abcdШ', 3148.29, new Buffer([0x12, 0x34]), '01234567-89AB-CDEF-0123-456789ABCDEF', 0.00000090000000000, new Date('December 4, 2011 10:04:23 GMT'), 'abcde'];
```

Comment:
Can you re-format this so the line is not so long? This'll also retrigger the flaky TravisCI tests and hopefully pass this time!

Thanks for adding the tests. We can merge this once the tests pass.